coalib/core/Bear.py

```diff
@@ -331,9 +331,18 @@ def check_prerequisites(cls):
         :return: True if prerequisites are satisfied, else False or a string
                  that serves a more detailed description of what's missing.
         """
+        def is_installed(requirement):
```

Comment:
Why not define on module level instead of creating a new function object for every call of `.check_prerequisites`?